serverless/integration/integration_test.go

```diff
@@ -46,7 +46,7 @@ func RunIntegration(t *testing.T, s log.Storage, f client.Fetcher, lh *rfc6962.H
 	ctx := context.Background()
 	lv := merkle.NewLogVerifier(lh)
 
-	// Do a few interations around the sequence/integrate loop;
+	// Do a few interactions around the sequence/integrate loop;
```

Comment:
Thanks for correcting my typo - I think it was intended to be "iterations", but "interactions" works too :)

Reply:
Oh, I just mindlessly went with lint's suggestion. Switched to "iterations"!